### PR TITLE
feat(addon-doc): improve code block

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Closes # 

## What is the new behaviour?

good example

image

----

image

image

image

### CODE REVIEW

File: projects/addon-doc/components/code/code.style.less [hunk 1]
```diff
@@ -24,26 +33,41 @@
     top: 0.75rem;
     right: 0.75rem;
     display: flex;
-    gap: 0.375rem;
     justify-content: center;
     align-items: center;
+    flex-direction: row-reverse;
+
+    @media @tui-mobile {
```
Comment (vladimirpotekhin) [inline projects/addon-doc/components/code/code.style.less:40]: nit: :host-context(tui-root._mobile) &

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/addon-doc/components/code/code.style.less (hunk 1 above)
Comment (splincode) [reply projects/addon-doc/components/code/code.style.less]: fixed

**Review by waterplea — COMMENTED** [review]
Maybe it can be `button tuiLink`?
image

File: projects/demo/src/modules/app/stackblitz/starter/stackblitz-edit-button.component.ts [hunk 2]
```diff
@@ -4,13 +4,15 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
...
             size="s"
             tuiButton
             type="button"
         >
-            Edit
+            <a tuiLink>
+                <tui-svg src="assets/icons/stackblitz.svg"></tui-svg>
+                Edit
+            </a>
         </button>
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/stackblitz/starter/stackblitz-edit-button.component.ts:16]: anchor inside button ? 🤔 

Should we just use the following code?
```html
<a
    icon="assets/icons/stackblitz.svg"
    iconAlign="left"
    tuiLink
>
    Edit
</a>
```

Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/starter/stackblitz-edit-button.component.ts]: fixed

Comment (waterplea) [reply projects/demo/src/modules/app/stackblitz/starter/stackblitz-edit-button.component.ts]: It shouldn't be `a`, it should be `<button tuiLink`, the link component can be applied to buttons too. `a` is a link while `button tuiLink` is a button that looks like a link. You cannot focus `a` without `href` and it is semantically wrong — what's why we made it apply to buttons too :)

Comment (splincode) [reply projects/demo/src/modules/app/stackblitz/starter/stackblitz-edit-button.component.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**